### PR TITLE
chore: added editor config

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

Adds editor config file to standardise spacing and lf

#### What changes did you make? (Give an overview)

added editor config file

#### Related Issues

regarding https://github.com/eslint/eslint.org/pull/461#pullrequestreview-1553580786

### CODE REVIEW

**Review by amareshsm — APPROVED** [review]
LGTM. Thanks! waiting for @kecrily  review.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
```
Comment (mdjermanovic) [inline .editorconfig:8]: It seems that the file breaks its own rule :)

**Review by mdjermanovic — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (harish-sethuraman) [reply .editorconfig]: 😂 I think this was because I did not have any formatter installed for `.editorconfig` files. Added one manually :p

**Review by snitin315 — APPROVED** [review]
LGTM, thanks!

**Review by kecrily — APPROVED** [review]
LGTM

**Pull request merged**